checker_test.go

```diff
@@ -118,6 +131,36 @@ got:
 want:
   nil
 `,
+}, {
+	about:   "Equals: error is not nil: not formatted",
+	checker: qt.Equals,
+	got: &errTest{
+		msg: "bad wolf",
+	},
+	args: []interface{}{nil},
+	expectedCheckFailure: `
+error:
+  got non-nil error
+got:
+  e"bad wolf"
+want:
+  nil
+`,
+}, {
+	about:   "Equals: error is not nil: not formatted and with quotes",
+	checker: qt.Equals,
+	got: &errTest{
+		msg: `failure: "bad wolf"`,
+	},
+	args: []interface{}{nil},
+	expectedCheckFailure: `
+error:
+  got non-nil error
+got:
+  e` + "`failure: \"bad wolf\"`" + `
```

Comment:
this is a little hard to read, but somewhat inevitable given backquote constraints I guess.
One possibility might be to do ``strings.Replace(test.expectedCheckFailure, "ò", "`")`` so we can have inline things that look a bit like backquotes without jumping through hoops. YMMV.

Reply:
Good suggestion, done (with tilde).